region/maridia/inner-yellow/Plasma Room.json

```diff
@@ -219,7 +219,8 @@
           "nodes": [1]
```

Comment:
space+plasma should be an acceptable option as well.

Reply:
Done.